Failed at the HplcSimulator@0.0.1 postinstall script 'sh ./install.sh'

`npm install` fails during post-install at `node node_modules/grunt-cli/bin/grunt build-prod --verbose` in install.sh:

> Running "uglify:js" (uglify) task
Verifying property uglify.js exists in config...OK
Files: dist/dev/js/hplc.js -> dist/prod/js/hplc.js
Options: banner="", footer="", compress, mangle=false, beautify=false, report="min", expression=false, maxLineLen=32000, ASCIIOnly=false
Minifying with UglifyJS...Reading dist/dev/js/hplc.js...OK
TypeError: Cannot create property 'warnings' on boolean 'true'
    at Object.exports.minify (/home/vmer/Git/hplc_simulator/node_modules/grunt-contrib-uglify/tasks/lib/uglify.js:82:35)
    at /home/vmer/Git/hplc_simulator/node_modules/grunt-contrib-uglify/tasks/uglify.js:136:25
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/home/vmer/Git/hplc_simulator/node_modules/grunt-contrib-uglify/tasks/uglify.js:61:16)
    at Object.<anonymous> (/home/vmer/Git/hplc_simulator/node_modules/grunt/lib/grunt/task.js:264:15)
    at Object.thisTask.fn (/home/vmer/Git/hplc_simulator/node_modules/grunt/lib/grunt/task.js:82:16)
    at Object.<anonymous> (/home/vmer/Git/hplc_simulator/node_modules/grunt/lib/util/task.js:301:30)
    at Task.runTaskFn (/home/vmer/Git/hplc_simulator/node_modules/grunt/lib/util/task.js:251:24)
    at Task.<anonymous> (/home/vmer/Git/hplc_simulator/node_modules/grunt/lib/util/task.js:300:12)
    at /home/vmer/Git/hplc_simulator/node_modules/grunt/lib/util/task.js:227:11
Uglifying source dist/dev/js/hplc.js failed.
Warning: Uglification failed.
Cannot create property 'warnings' on boolean 'true'. 
Use --force to continue.

> Aborted due to warnings.

>npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.3
npm ERR! Linux 4.15.0-20-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! HplcSimulator@0.0.1 postinstall: `sh ./install.sh`
npm ERR! Exit status 6
npm ERR! 
npm ERR! Failed at the HplcSimulator@0.0.1 postinstall script 'sh ./install.sh'.

[npm-debug.log](https://github.com/mlibby/hplc_simulator/files/1976418/npm-debug.log)

--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -146,7 +146,7 @@ module.exports = function(grunt) {
     uglify: {
       options: {
         mangle: false,
-        compress: true,
+        compress: {},
         beautify: false
       },
       js: {

